api/main.py

```
@@ -51,7 +52,19 @@ async def lifespan(app: FastAPI):
+    # their connection is still established), then follow the sshd log for
+    # live connect/disconnect events.
+    await close_stale_open_sessions()
+    tracker_task = asyncio.create_task(follow_sshd_log())
+
     yield
+
+    tracker_task.cancel()
+    try:
+        await tracker_task
+    except asyncio.CancelledError:
+        pass
```

Ruff (0.15.21)

[warning] Use `contextlib.suppress(asyncio.CancelledError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(asyncio.CancelledError): ...`

(SIM105)